src/Raven.Client/Json/Serialization/NewtonsoftJson/Internal/DefaultRavenSerializationBinder.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json.Serialization;
+
+namespace Raven.Client.Json.Serialization.NewtonsoftJson.Internal;
+
+internal class DefaultRavenSerializationBinder : DefaultSerializationBinder
```

Comment:
We can make this public, and then users can directly add to `SafeTypes`, etc if they need to.
Easier than creating their own serialization binders.

Reply:
Done, can you check if this would work?